src/adapters/parcel-rights-fetcher.ts

```diff
@@ -17,18 +18,31 @@ const QUERY_OPERATORS_PARCEL: string = `
       address
     }
     updateOperator
+    operator
   }
   estates(
     where: { parcels_: { x: $x, y: $y } }
   ) {
     owner {
       address
     }
+    operator
```

Comment:
just a personal toc 😆  but can we have them in the same order?

`updateOperator`
`operator`
or

`operator`
`updateOperator`

Reply:
Changed it!